### ISSUE TITLE
Custom Logging

### ISSUE THREAD

**Issue by ghomasHudson** [PyTorchLightning/lightning-transformers#178]
## 🚀 Feature
This may already exist, but it would be great to be able to write plugins for different experiment logging methods, e.g. wandb (weights and biases), tensorboard, etc... 

### Motivation

For many ML experiments, something more organized than log files is useful, especially for metrics such as the loss, which are best graphed.

### Pitch

Would love to be able to add something like `python train.py log=wandb`, (though maybe more options are needed so `log.type="wandb"` is better)

**Comment by SeanNaren** [comment]
I think this has been added in #177 :)

**Comment by ghomasHudson** [comment]
> I think this has been added in #177 :)

Amazing! This project is moving super fast now.
